inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/special/ClassLoaderDelegation.java

```diff
@@ -64,6 +64,10 @@ public boolean requiresInstrumentationChange(Class<?> clazz, InstrumentationConf
 
     @Override
     public DynamicType.Builder instrument(Class<?> clazz, InstrumentationConfiguration settings, DynamicType.Builder builder) {
+
+        //ensure that this bootstrap class is loaded BEFORE any classloader is instrumented
+        ClassLoaderDelegationMarker.CLASS_LOADER_DELEGATION_PERFORMED.get();
```

Comment:
Can you move this to the static initializer? Imo it is cleaner and definitly loaded before any instrumentation. Or would this do any problems?

```
static {
        ClassLoaderDelegationMarker.CLASS_LOADER_DELEGATION_PERFORMED.get();
}
```